Library/Formula/vc4asm.rb

```diff
@@ -0,0 +1,16 @@
+class Vc4asm < Formula
+  homepage "http://maazl.de/project/vc4asm/doc/index.html"
+  url "https://github.com/maazl/vc4asm/archive/V0.1.7.tar.gz"
+  sha1 "20594439e9011dbd2175903754735cafbd01f029"
+
+  def install
+    rm_rf "bin"
+    mkdir "bin"
+    mkdir "obj"
```

Comment:
It'd be nice to get even a simple PR upstream to fix it not making a directory it needs to compile. Presuming they accepted it we could add it as a patch before the next release.